src/components/farmModal.html

```diff
@@ -195,13 +195,13 @@ <h4><u>Farm</u></h4>
                     <br/>Once fully grown, the Berry plant can be clicked to manually harvest the Berries and gain some Farm Points.</p>
 
                     <p>Otherwise after some time being fully grown, the Berry plant will wither away,
-                    dropping only a fraction of the Berries but may also be re-planted automatically.
+                    dropping half of the expected Berries but may also be re-planted automatically.
                     <br/><i>(Farm Points are only gained by manually harvesting Berries)</i><p>
 
                     <p>Additional plots can be unlocked by spending Berries.</p>
 
                     <h4><u>Mutations</u></h4>
-                    <p>Berries have the chance to mutate into other Berries based on certain conditions.
+                    <p>Berries have the chance to cause mutations and create new Berry species based on certain conditions.
```

Comment:
This seems to also be a common source of confusion:
```suggestion
                    <p>Berries have the chance to cause mutations and create new Berry species in empty plots based on certain conditions.
```

Reply:
Only issue is that some mutations don't require empty plots. Not sure if we want that distinction in the help tab though.

Reply:
Maybe in the hint text for those few we can specify that it will actually change the berry?

Reply:
I'm okay with changing it to "in empty plots". Hopefully that doesn't confuse the player later on with evolve mutations though.